api/utils/slices.go

```diff
@@ -20,51 +20,6 @@ import (
 	"strings"
 )
 
-// CopyByteSlice returns a copy of the byte slice.
-func CopyByteSlice(in []byte) []byte {
```

Comment:
I am not too concerned about it. We really never should have had this in a public API anyway and API consumers are used to pinning the API module by now since we've never supported proper versioning.

Reply:
Fair enough.